packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -967,6 +972,14 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             metadata,
         );
 
+        if (this.garbageCollector.shouldRunSweep) {
+            const expiryMs = this.runtimeOptions.gcOptions?.gcSessionExpiryTime ?? defaultContainerRuntimeExpiryMs;
+            this.sessionExpiryTimeout = setTimeout(() => this.closeFn({
```

Comment:
"ClientSessionExpired" please add it to ContainerErrorType (with good comment on when and why this error is thrown) such that host developers know the need to deal with it and know what kind of message to craft to show to user. The message you have is for developers only, as it's not localized, and likely hosts want to have control over actual message shown to user.

I'd also use one of the pre-canned classes, not just object. We put a bunch of fields into those objects to ensure we can track where error was thrown and if it's ours or not. Likely you need to create your own - please see UsageError() for an example.